internal/table/client.go

```diff
@@ -190,39 +192,28 @@ func (c *client) createSession(ctx context.Context) (s Session, err error) {
 		}()
```

Comment:
it is safer to call defer close() closer to the beginning of the goroutine

Reply:
ack